test/functional/test-viewer.js

```diff
@@ -432,6 +432,13 @@ describe('Viewer', () => {
     expect(m.data.height).to.equal(14);
   });
 
+  it('should post scroll event', () => {
```

Comment:
Ideally you'd add a test that emits multiple scroll events, ticks an animation frame, and tests the throttling.